docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -46,7 +46,7 @@ This guide applies to Intel Core Ultra and Core 12 - 14 gen integrated GPUs, as
 
 * With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU: 
   ```bash
-  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+  pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/cn/
```

Comment:
Need to add both links (us & cn)

Reply:
@ivy-lv11 please help verify if `pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/` works.